### ISSUE TITLE
It breaks a fresh ember installation

### ISSUE THREAD

**Issue by mariusleu** [auth0-community/ember-simple-auth-auth0#129]
Steps to reproduce:

```
ember new my-app
cd my-app
ember install ember-simple-auth
ember install ember-simple-auth-auth0
ember s
```
Navigate to localhost:4200, you will see the following error in the console:
```
vendor.js:62487 Uncaught SyntaxError: Unexpected identifier
app.js:1 Uncaught ReferenceError: define is not defined
    at app.js:1
```
vendor.js:62487:
````
	import Authentication from './authentication';
````
more lines from vendor.js:

````
/* 0 */
/***/ (function(module, exports, __webpack_require__) {

	module.exports = __webpack_require__(1);


/***/ }),
/* 1 */
/***/ (function(module, exports) {

	import Authentication from './authentication';
	import Management from './management';
	import WebAuth from './web-auth';
	import version from './version';

	export { Authentication, Management, WebAuth, version };

	export default {
	  Authentication: Authentication,
	  Management: Management,
	  WebAuth: WebAuth,
	  version: version
	};


/***/ })
/******/ ])
});
````

I've been using this library successfully in older ember projects, but now it breaks the new ember app. Tried on both ember v2.18.2 and v3.x.x

**Comment by mariusleu** [comment]
Update:

Added the following in "resolutions" in package.json
````
    "ember-simple-auth-auth0/auth0-lock": "11.0.1"
```

With this specific version of auth0-lock my app works.

**Comment by Kilowhisky** [comment]
This is not working for me on my brand new project running ember 3.3.0.

This is what i get when i try the "resolutions" approach.
`warning Resolution field "auth0-lock@11.0.1" is incompatible with requested version "auth0-lock@^11.4.0"`

I'm still getting

```
vendor.js:62503 Uncaught SyntaxError: Unexpected identifier
app.js:1 Uncaught ReferenceError: define is not defined
    at app.js:1
```

**Comment by XaserAcheron** [comment]
Heh. "define is not defined." That's the goofiest error message I've seen in a while. :P

Anyhow, it looks like something in webpack is goofing up on newer versions of auth0.js/lock. I'm not 100% sure what it is, but fortunately we just switched away from webpack in favor of `ember-auto-import` for pulling in the Auth0 dependencies, so I think this is already fixed on the `develop` branch.

I created a couple of test projects to verify:
- Made a new app, installed `ember-simple-auth` and `ember-simple-auth-auth0`; it broke, just as described.
- Made a 2nd new app, installed `ember-simple-auth` and `https://github.com/auth0-community/ember-simple-auth-auth0.git#develop`, and it works fine. Whew!

Just need to cut a new release with the fix, I suppose.
